awx_collection/plugins/modules/tower_user.py

```diff
@@ -55,6 +55,14 @@
       description:
         - Write-only field used to change the password.
       type: str
+    update_password:
```

Comment:
I'm all in favor of getting this merged, but to be transparent I will state my only remaining hesitation.

We have other modules which take secrets, like `tower_credential` will accept secrets inside of the `inputs` field. It's possible that we could move this logic to the shared code in module_utils. For that, it would make more sense to have a general setting like `update_secrets` so there wouldn't be different parameters for each secret-ish field.

We can change the implementation later, but we can't rename the parameter as easily.

Reply:
IMHO an `update_secrets` parameter would only make sense if we rename `password` to `secret`. Once the `tower_user` module implements a `password` parameter, I think the best approach is to name this new one `update_password`.

As an example, [`user`](https://docs.ansible.com/ansible/latest/modules/user_module.html#parameter-password) module implements the parameters `password`/`update_password` and [`win_credential`](https://docs.ansible.com/ansible/latest/modules/win_credential_module.html#parameter-secret) implements the parameters `secret`/`update_secret`. But they all remain consistent in their names. 

As for whether to move it to module_utils, this can always be done later, if necessary.